modules/jenkins_files/files/var/lib/jenkins/hudson.plugins.warnings.WarningsPublisher.xml

```diff
@@ -3,17 +3,23 @@
   <groovyParsers>
     <hudson.plugins.warnings.GroovyParser>
       <name>CMake</name>
-      <regexp>^CMake (Deprecation Warning|Warning|Warning \(dev\)|Error) at (.+):(\d+)( \((.+)\))?:\n(  .+\n)?(.+\n(  .+\n)*)?</regexp>
+      <regexp>^CMake (Deprecation Warning|Warning|Warning \(dev\)|Error)( at (.+):(\d+)( \((.+)\))?)?:\n(?!  Manually-specified variables were not used by the project:)(  .+\n)?(.+\n(  .+\n)*)?</regexp>
```

Comment:
We should consider updating the regexp to use named capture groups in a follow-up pull request to avoid the index shuffling below.